test/unit/MerkleDrop.spec.js

```diff
@@ -71,9 +71,9 @@ const setupInitialState = async (contractInstances, initialState) => {
   const {
     withProof,
     trancheExpired,
-    forwardBlocks,
     zeroAllocation,
     incorrectProof,
+    withoutSeededAllocation,
   } = initialState;
 
   let parsedAllocations = matchAllocationsWithAddresses(
```

Comment:
Btw, why do we need `parsedAllocations` and `cumulativeAllocation` as variables? Is it possible to use `const` instead?

Reply:
`parsedAllocations` can be made a constant. The other variable needs to remain overwritable for one test case. Thanks for spotting!

Reply:
Roger, great